analyzer/src/funTest/assets/projects/synthetic/gradle-bom-expected-output.yml

```diff
@@ -18,140 +18,26 @@ project:
   scopes:
```

Comment:
Commit message:
* The placeholder "1234567" probably needs to be replaced by an actual hash.
* In the last paragraph: "issued" -> "issue"

Reply:
Also:
- Comma after "Before this change"
- "would in turns" -> "would in turn"